scripts/ralph-loop.sh

```
@@ -0,0 +1,896 @@
+
+git pull origin "$BASE_BRANCH" 2>/dev/null || true
+
+# --- Fetch issues function ---------------------------------------------------
+fetch_issues() {
+  CANNY_POSTS=$(curl -s -X POST https://canny.io/api/v1/posts/list \
+    -d "apiKey=${CANNY_API_KEY}" \
+    -d "boardID=${CANNY_BOARD_ID}" \
+    -d "sort=score" \
+    -d "limit=30")
+
+  local tmp_canny=$(mktemp)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)